python/orca/src/bigdl/orca/data/image/voc_dataset.py

```diff
@@ -62,14 +62,16 @@ def __init__(self, root: str="VOCdevkit",
         self._im_cache = {}  # type: Dict[str, "ndarray"]
 
     def _load_items(self, splits_names: List[Tuple[int, str]]) -> List[Tuple[str, str]]:
-
         img_ids = []
+        img_id_allow_list = list(range(1000000))
```

Comment:
Remark: the files will at most be six digits and should be smaller than this range.